src/App.css

```diff
@@ -1,51 +1,60 @@
 .App {
-  text-align: center;
+    text-align: center;
 }
 
 .App-logo {
-  height: 40vmin;
-  pointer-events: none;
+    height: 40vmin;
+    pointer-events: none;
+}
+
+.routes_height {
+    min-height: 75vh;
+}
+
+.link_ {
+    text-decoration: none;
+    color: inherit;
 }
 
 @media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+    .App-logo {
+        animation: App-logo-spin infinite 20s linear;
+    }
 }
 
 .App-header {
-  background-color: #22201c;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  color: white;
+    background-color: #22201c;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    color: white;
 }
 
 .App-link {
-  color: #61dafb;
+    color: #61dafb;
 }
 
 .brand-title {
-  font-size: 3rem;
-  color: #d9d9d9;
+    font-size: 3rem;
+    color: #d9d9d9;
 }
 
 .brand-title span {
-  color: #ffde59;
+    color: #ffde59;
 }
 
 .brand-description {
-  font-size: 1.4rem;
+    font-size: 1.4rem;
 }
 
 .links {
-  display: flex;
+    display: flex;
 }
 
 .links a {
-  color: #d9d9d9;
-  font-size: 1.2rem;
-  margin: 1rem;
-}
+    color: #d9d9d9;
```

Comment:
You can use variable for this color as you have used it in the navbar and if you are going to use it in the footer.

Reply:
i will consider this suggestion in my next pr